debian/copyright

```diff
@@ -87,3 +83,103 @@ License: GPL-2+
+ .
+ Python was created in the early 1990s by Guido van Rossum at Stichting
+ Mathematisch Centrum (CWI, see http://www.cwi.nl) in the Netherlands
+ as a successor of a language called ABC.  Guido remains Python's
+ principal author, although it includes many contributions from others.
+ .
+ In 1995, Guido continued his work on Python at the Corporation for
+ National Research Initiatives (CNRI, see http://www.cnri.reston.va.us)
+ in Reston, Virginia where he released several versions of the
+ software.
+ .
+ In May 2000, Guido and the Python core development team moved to
+ BeOpen.com to form the BeOpen PythonLabs team.  In October of the same
+ year, the PythonLabs team moved to Digital Creations, which became
+ Zope Corporation.  In 2001, the Python Software Foundation (PSF, see
+ https://www.python.org/psf/) was formed, a non-profit organization
+ created specifically to own Python-related Intellectual Property.
+ Zope Corporation was a sponsoring member of the PSF.
+ .
+ All Python releases are Open Source (see http://www.opensource.org for
+ the Open Source Definition).  Historically, most, but not all, Python
+ releases have also been GPL-compatible; the table below summarizes
+ the various releases.
+ .
+     Release         Derived     Year        Owner       GPL-
+                     from                                compatible? (1)
+ .
+     0.9.0 thru 1.2              1991-1995   CWI         yes
+     1.3 thru 1.5.2  1.2         1995-1999   CNRI        yes
+     1.6             1.5.2       2000        CNRI        no
+     2.0             1.6         2000        BeOpen.com  no
+     1.6.1           1.6         2001        CNRI        yes (2)
+     2.1             2.0+1.6.1   2001        PSF         no
+     2.0.1           2.0+1.6.1   2001        PSF         yes
+     2.1.1           2.1+2.0.1   2001        PSF         yes
+     2.1.2           2.1.1       2002        PSF         yes
+     2.1.3           2.1.2       2002        PSF         yes
+     2.2 and above   2.1.1       2001-now    PSF         yes
+ .
+ Footnotes:
+ .
+ (1) GPL-compatible doesn't mean that we're distributing Python under
+     the GPL.  All Python licenses, unlike the GPL, let you distribute
+     a modified version without making your changes open source.  The
+     GPL-compatible licenses make it possible to combine Python with
+     other software that is released under the GPL; the others don't.
+ .
+ (2) According to Richard Stallman, 1.6.1 is not GPL-compatible,
+     because its license has a choice of law clause.  According to
+     CNRI, however, Stallman's lawyer has told CNRI's lawyer that 1.6.1
+     is "not incompatible" with the GPL.
+ .
+ Thanks to the many outside volunteers who have worked under Guido's
+ direction to make these releases possible.
+ .
+ .
+ B. TERMS AND CONDITIONS FOR ACCESSING OR OTHERWISE USING PYTHON
+ ===============================================================
+ .
+ PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
```

Comment:
Don't you think the license text below this header would be enough?

Reply:
In a good majority of python packages in debian the text is complete. For exactness, it should stay complete. That is why I dropped it to the bottom of the file, to get it out of the way.

Reply:
Fine with me :-)